config/config/sc-config.yaml

```diff
@@ -1092,3 +1112,19 @@ networkPolicies:
       # IP to trivy vulnerability database
       egressIPs:
         - "0.0.0.0/0"
+  monitoring:
+    enabled: true
+    grafana:
+      externalDashboardProvider: # loading dashboards from grafana website
+        ips:
+          - "0.0.0.0/0"
```

Comment:
It can be good to have "set-me" everywhere for ips. Even if most of them will always be "0.0.0.0/0". It can be good to always that that decision.

Reply:
Do we want that even for these more external things that might be rather unstable?
It does make it a more conscious decision to open it up, but it will also clutter the config a bit.

Reply:
After some discussion on slack we decided on changing to "set-me"